src/en/space-station-14/departments/science/proposals/scientific-research-proposal.md

```diff
@@ -0,0 +1,35 @@
+-	Many research items seem to be under costed with respect to the current ease of points generation. This leads to shifts where all research projects can be readily completed, particularly during longer than normal shifts and / or shifts that have a large research team.
+-	Once all research projects have been completed, there is no longer any compelling reason to conduct any further research.
+
+## Proposed changes
+The changes to scientific research outlined in this proposal have been devised in an attempt to alleviate the issues listed above. These changes include:
+-	Making all low-tier / filler researchable items available to print at round start. These include items such as proximity sensors, TV circuit boards, and surveillance camera equipment.
+-	Making the more desirable mid-tier researchable items (e.g., RCDs, lasers, and high-capacity batteries) also available to print at round start, but with the caveat that science points are required as part of their construction. Lore-wise, this can be explained as the technology already being widely known, however, you need to purchase technology licenses from NT in order to print them. 
+-	Scientists can print technology licenses from the tech disk terminal, exchanging their research points for discipline specific licenses (e.g., a single civilian technology license might cost 150 research points, a single industrial technology license 250, and a single arsenal technology license 350).
+-	These licenses can then be fed into (or ejected from) an appropriate lathe just like any other resource material (e.g., arsenal technology licenses can be inserted into security and ammo fabs, civilian technology licenses into medical fabs, and experimental technology licenses into protolathes).
+-	Licensed items require a number of licenses that correspond to their relative power / usefulness (e.g., a laser carbine might require five licenses to print, in addition to their other base materials, while a box of incendiary ammunition might only need one license). 
+-	The highest tier / most powerful technologies (e.g., bluespace backpacks, speed boots, teleporter pads; perhaps a dozen technologies all up) would still need to be researched the traditional way, requiring an expenditure of research points at the R&D computer to unlock (in addition to the license cost to print them afterward; unfortunately, NT owns all patentable technology produced by their research staff). However, each of these research projects now requires a vast amount of research points to unlock, in the range of hundreds of thousands. The ultimate goal is that unlocking a new technology should be a relatively rare occurrence within the shift, with perhaps only two or three unlocks per shift being common.
+
```

Comment:
Its important to add, that a lot of researches are 'schema/board' only, and to roll with this new scheme those boards should be now available at respective fabs. Which leaves 'board printer' device kinda useless. We can just add every board into it, or remove it. I'm not sure, honestly, which is better.

Reply:
Do you mean the circuit imprinter? I mean. there's no reason why it couldn't print licensed boards as well, or just be the sole source of boards

Reply:
Your answer does not contain any clue what EXACTLY should be line of separation - which stuff do what. At least conceptually.
I suppose you would like to leave 'circuit printer' to have just have all circuit boards as it works right now. If so - i'd propose mentioning in proposal, that boards from recepies are going to be available from respective department fabs AND ALSO from circuit printer - as it is now.